common/djangoapps/student/tests/test_certificates.py

```diff
@@ -22,6 +24,9 @@
 
 # pylint: disable=no-member
 
+YESTERDAY = datetime.datetime.now(UTC) - datetime.timedelta(days=1)
+TOMORROW = datetime.datetime.now(UTC) + datetime.timedelta(days=1)
```

Comment:
If the idea with these is to make sure that they're in the past/future, it's probably safest to use a delta of 2 days. `YESTERDAY` doesn't have this concern, but `TOMORROW` could be set at 11:59:59 pm during a specific test run, then one second later the value of `TOMORROW` is actually today.